doc/release-notes-27501.md

```diff
@@ -0,0 +1,3 @@
+- A new `getprioritisationmap` RPC has been added. It returns a map of all fee deltas created by the
```

Comment:
`map` is a C++ concept, this is returning a json object. Wouldn't avoiding the implementation baggage by calling it something like `getprioritisetransactions` be better?

Reply:
Good point, changed to `getprioritisedtransactions`